sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/TrueRuleFilter.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.administration.models;
+
+/**
+ * Matches all the messages arriving to be selected for the subscription.
+ */
+public final class TrueRuleFilter extends SqlRuleFilter {
+    private static final TrueRuleFilter INSTANCE = new TrueRuleFilter();
```

Comment:
Did we consider not having these types , they are String sqlExpression and user can directly create instance of SqlRuleFilter ?

Reply:
They exist in the .NET API and I believe we talked about this.